cardano-cli/src/Cardano/CLI/Orphan.hs

```diff
@@ -49,3 +57,22 @@ instance ToJSON HashableScriptData where
       [ "hash" .= hashScriptDataBytes hsd
       , "json" .= scriptDataToJsonDetailedSchema hsd
       ]
+
+instance
```

Comment:
This looks odd, but I understand that's for the transitionary period until we go switch completely to just throwing exceptions in CLI - do I understand this right?

If that's the case, can you leave a TODO to remove this instance later?

Reply:
Yes it should be transitionary. It comes from `readVerificationKeySource`.